src/main/java/io/reactivex/annotations/SchedulerSupport.java

```diff
@@ -28,6 +28,7 @@
 @Retention(RetentionPolicy.RUNTIME)
 @Documented
 @Target({ElementType.METHOD, ElementType.TYPE})
+@SuppressWarnings("unused") // Part of Public API.
 public @interface SchedulerSupport {
```

Comment:
Well `IO` & `NEW_THREAD` are unused since they are not used internally. However they're part of the public API so I suppressed that warning that they're not used. Why will Eclipse complain about this one?